tests/cppunit/types/set_test.cc

```diff
@@ -191,6 +191,23 @@ TEST_F(RedisSetTest, Inter) {
   set_->Del(k3);
 }
 
+TEST_F(RedisSetTest, InterCard) {
```

Comment:
Additionally, you can add a test for a corner/edge case when one of the sets is empty. And one for the case when there are no common elements in both sets.

Reply:
I add these in `inter` tests later beacause `intercard` is a subset of calling `inter` .